src/main/java/com/welovecoding/nbeditorconfig/listener/ListenerAttacher.java

```diff
@@ -0,0 +1,50 @@
+package com.welovecoding.nbeditorconfig.listener;
+
+import com.welovecoding.nbeditorconfig.config.Settings;
+import com.welovecoding.nbeditorconfig.processor.SmartSkip;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.netbeans.api.project.Project;
+import org.openide.filesystems.FileObject;
+
+/**
+ *
+ * @author Michael Koppen
```

Comment:
I think we can skip all these `@author` comment skeletons. They take 4 lines of code for no valuable information. You can see our contributors on GitHub and in `git blame`.